android/expoview/src/main/java/host/exp/exponent/experience/HomeActivity.java

```diff
@@ -143,6 +144,7 @@ protected void onError(final Intent intent) {
         new CameraPackage(),
         new FaceDetectorPackage(),
         new MediaLibraryPackage(),
+        new NotificationsPackage(),
```

Comment:
It's funny, cause when we start `home`, it loads `ExpoModuleRegistryAdapter`. Then, it starts initialized `expo-notifications`... and this will fail. Cause singleton isn't created.

Maybe, I should check if the singleton exists here: https://github.com/expo/expo/blob/47bed0d74eedf6067735c789fda713c0857df467/android/expoview/src/main/java/versioned/host/exp/exponent/modules/universal/notifications/ScopedNotificationsEmitter.java#L25-L28

Reply:
I decided to leave it as it is for now. We can fix it in the future. Maybe even in this SDK if we finished this integration process.